### PR TITLE
Fetch data for the salary table

### LINKED ISSUE #51: Salary Totals page (data fetching)
Implement data fetching for the Salary page:
- aggregate all shifts worked this month
- calculate the income for each person
  - fetch daily income of the bar
  - add regular salary and bonuses
  - fetch deductions for each person

### PR DESCRIPTION
**Pull request by ANDREYDEN**
## Description

- [x] Display **aggregated salary** based on shifts
- [x] Display **aggregated deductions** from Poster
- [x] Display **sales income** based on shifts and data from Poster
- [x] Display **bonuses**

## Issue

Fixes #51

### CODE REVIEW

File: lib/services/poster/posterService.ts [hunk 1]
```diff
@@ -9,28 +12,83 @@ export const posterInstance = axios.create({
...
+    }
+
+    const salesIncomeTotals: EmployeesMonthlyStatDto = {}
+
+    for (const employee of employees) {
+        salesIncomeTotals[employee.id] = sales.data
+            .reduce(
+                (total, dateSales, date) => {
+                    const currentDay = dayjs().set('date', date + 1)
+                    console.log({ e: employee.first_name, date: date + 1, shifts: shifts.filter(shift => employee.id === shift.employee_id) })
```
Comment (alisondraV) [inline lib/services/poster/posterService.ts:78]: Leftover?

File: lib/services/poster/posterService.ts [hunk 2]
```diff
@@ -9,28 +12,83 @@ export const posterInstance = axios.create({
...
+            salesIncomeTotalsError 
+        }
+    }
+
+    const salesIncomeTotals: EmployeesMonthlyStatDto = {}
+
+    for (const employee of employees) {
+        salesIncomeTotals[employee.id] = sales.data
+            .reduce(
+                (total, dateSales, date) => {
```
Comment (alisondraV) [inline lib/services/poster/posterService.ts:76]: Inline

Comment (ANDREYDEN) [reply lib/services/poster/posterService.ts:76]: no?

Comment (ANDREYDEN) [reply lib/services/poster/posterService.ts:76]: no?

Comment (alisondraV) [reply lib/services/poster/posterService.ts:76]: :(

**Review by alisondraV — APPROVED** [review]

**Pull request merged**